### PR TITLE
CLN:Added annotations to functions

### PR DESCRIPTION
**Pull request by ShaharNaveh**
- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] passes `black pandas`
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: pandas/_libs/tslibs/timestamps.pyx [hunk 1]
```diff
@@ -657,7 +657,7 @@ timedelta}, default 'raise'
         return ccalendar.get_day_of_year(self.year, self.month, self.day)
 
     @property
-    def week(self):
+    def week(self) -> int:
```
Comment (MomIsBestFriend) [inline pandas/_libs/tslibs/timestamps.pyx]: Not sure if this is correct. hope that one of the developers will correct me if needed.

File: pandas/_libs/tslibs/timestamps.pyx [hunk 2]
```diff
@@ -666,7 +666,7 @@ timedelta}, default 'raise'
     weekofyear = week
 
     @property
-    def quarter(self):
+    def quarter(self) -> int:
```
Comment (MomIsBestFriend) [inline pandas/_libs/tslibs/timestamps.pyx]: Same here.

File: pandas/_libs/tslibs/timestamps.pyx (hunk 1 above)
Comment (jbrockmendel) [reply pandas/_libs/tslibs/timestamps.pyx]: this is correct

File: pandas/_libs/tslibs/timestamps.pyx (hunk 2 above)
Comment (jbrockmendel) [reply pandas/_libs/tslibs/timestamps.pyx]: also correct

File: pandas/_libs/tslibs/timestamps.pyx (hunk 1 above)
Comment (MomIsBestFriend) [reply pandas/_libs/tslibs/timestamps.pyx]: thank you @jbrockmendel

**Comment by jbrockmendel** [comment]
im not sure if we have a convention for `__setstate__` or `strptime` cases, the rest look good

**Comment by MomIsBestFriend** [comment]
> im not sure if we have a convention for `__setstate__` or `strptime` cases, the rest look good

Would you like me to remove them, at least for this PR. and open a separate one containing only them?

**Comment by jbrockmendel** [comment]
> Would you like me to remove them, at least for this PR. and open a separate one containing only them?

Either that or wait for @WillAyd to chime in

**Comment by jbrockmendel** [comment]
Thanks @MomIsBestFriend

**Pull request merged**